### ISSUE TITLE
Will no longer show amount of days until holiday

### ISSUE THREAD

**Issue by Pvtjoker77** [pinkywafer/Calendarific#35]
It pretty much stopped working at the beginning of the year. Will no longer show you how many days until the next holiday so you can't even use it in an automation any longer. It was in my opinion, the best integration for holiday lighting I had ever seen, and I hope it can get fixed. I've been trying everything to try to make it work again. Changing my API key after I sent this since it's included in one of the pictures.
<img width="622" alt="Screenshot 2023-02-15 at 12 21 05 PM" src="https://user-images.githubusercontent.com/51425835/219119078-d4b52153-bdf1-41a0-bf82-7bf33c393446.png">
<img width="565" alt="Screenshot 2023-02-15 at 12 20 04 PM" src="https://user-images.githubusercontent.com/51425835/219119083-102782f1-bd87-4603-83db-5feb6aace50e.png">
<img width="568" alt="Screenshot 2023-02-15 at 12 19 23 PM" src="https://user-images.githubusercontent.com/51425835/219119084-7486fe64-e369-45b6-a260-6fd678c8b75f.png">
<img width="474" alt="Screenshot 2023-02-15 at 12 19 04 PM" src="https://user-images.githubusercontent.com/51425835/219119086-36513e7d-bee8-4542-beba-512276681038.png">

**Comment by pinkywafer** [comment]
anything in the log?  I'm guessing that the calendarific api has hit the rate limit - I've seen it happen a few times for no fathomable reason... and it doesn't seem to reset and I was unable to get hold of the Calendarific team the last time I tried.  If so, a new sign up (and so new api key) should cure it

**Comment by Pvtjoker77** [comment]
That's exactly what it is. I created a new account and got a new API and it's working. When I logged in to my calendarific account it was only showing I used 8% so that's why I didn't think it was the problem but now it's working once I got a new account. Thank you.
<img width="499" alt="Screenshot 2023-02-16 at 4 13 01 PM" src="https://user-images.githubusercontent.com/51425835/219499030-0aedb022-24ae-4402-92ae-795b44b49a7a.png">
